charts/sysdig/templates/_helpers.tpl

```diff
@@ -68,7 +68,7 @@ Return the proper Sysdig Agent image name
 {{- if .Values.image.overrideValue }}
     {{- printf .Values.image.overrideValue -}}
 {{- else -}}
-    {{- include "sysdig.imageRegistry" . -}} / {{- include "sysdig.repositoryName" . -}} : {{- .Values.image.tag -}}
+    {{- include "sysdig.imageRegistry" . -}} / {{- include "sysdig.repositoryName" . -}} : {{- .Values.image.tag -}} {{- if .Values.image.digest -}} @ {{- .Values.image.digest -}} {{- end -}}
```

Comment:
@KReg27 can we use only the digest when it's available, rather than tag and digest together? Since the digest will be given preference anyway, is there any benefit to keeping both?

Reply:
@achandras No specific reason. I have made the change as suggested.